src/common/common.js

```diff
@@ -57,16 +57,26 @@ const common = {
     const result = new BigNumber(satoshi).div('1e8');
     return result;
   },
-  rskCoinToWeiHex(amount) {
-    const result = `0x${this.rskCoinToWei(amount).decimalPlaces(0).toString(16)}`;
+  rskCoinToWeiHex(amount, precision) {
```

Comment:
Do we really need 2 functions for decimal numbers and hex numbers?

The best ways should be using rskCoinToWei() to return a big number, then you can use rsk3.utils functions to get hex or decimal number.